utils/dynamodb/formatResponse.js

```diff
@@ -0,0 +1,162 @@
+import { 
+  DYNAMO_DB_ATTRIBUTE_TYPES, 
+  DYNAMO_DB_ORDER, 
+  DYNAMO_DB_TYPES 
+} from "@/constants";
+
+/** 
+ * RESPONSE FORMATTER 
+ * */
+
+// # 1: DATA FORMATTER
+// Formats rawEntityTypeSlug response
+// This function will only return the values that are included in the attributes of the content_type
+// This function automatically formats the value based on its type
+export const formatEntityTypeSlugResponse = (data, attributeMetadata) => {
+  return data.Items?.reduce((result, item) => {
+    const itemKey = item.id.S;
+
+    const defaultItems = {
+      id: itemKey,
+      slug: item.slug.S,
+      status: item.status.S,
+      order: parseInt(item.order.N),
+    };
+
+    if (item.values) {
+      const values = Object.keys(item.values.M).reduce((updatedAttrs, attributeName) => {
+        if (attributeMetadata[attributeName]) {
+          const attribute = item.values.M[attributeName];
+          const attributeType = attributeMetadata[attributeName].type;
+
+          // formats the value based on type
+          updatedAttrs[attributeName] = formatValueByType(attributeType, attribute[Object.keys(attribute)[0]]);
+        }
+
+        return updatedAttrs;
+      }, {});
+
+      result[itemKey] = {
+        ...defaultItems,
+        ...values,
+      };
+
+    } else {
+      result[itemKey] = defaultItems;
+    }
+
+    return result;
+  }, {});
+};
+
+// # 2: ATTRIBUTES FORMATTER
+// Formats the attributes from the content_type
+// Usese the formatAttributeMap function to map the attributes correctly
+export const formatAttributesData = (data) => {
+  const attributes = data
+    .reduce((result, item) => {
+      if (item.attributes) {
+        const attributesMap = item.attributes.M;
+        result = formatAttributeMap(attributesMap);
+      }
+      return result;
+    }, {});
+  
+  return attributes;
+}
+
+/** 
+ * ATTRIBUTE TYPE FORMATTER 
+ * */
+
+// # 1: IMAGE ATTRIBUTE TYPE FORMATTER
+// Formats image to its correct form 
+// returns an object { key, name, link }
+// This is based on how images are originally formatted in KlaudSol CMS
+export const formatImage = (image) => {
+  const imageUrl = process.env.KS_S3_BASE_URL;
+  let originalName = '';
+  if (image.includes("_")) {
+    originalName = image.split("_")[1];
```

Comment:
```suggestion
/* 
this is how we generate the uniqueKey (file name of image / video). The key is the actual file name of the uploaded image (e.g. geli.png). It will be appended by a random value separated by a comma. As a result, the file name uploaded to the s3 and saved to the database follows this format. (e.g. 0xlofisl_geli.png);
*/

  if (image.includes("_")) {
    originalName = image.split("_")[1];
```